.gitignore

```diff
@@ -0,0 +1,2 @@
+Cargo.lock
+target/
```

Comment:
I had some trouble checking files in; this made it a bit easier. Hope it's okay I added this.